server_addon/create_ayon_addons.py

```diff
@@ -303,6 +322,54 @@ def main(create_zip=True, keep_source=False):
 
 
 if __name__ == "__main__":
-    create_zip = "--skip-zip" not in sys.argv
-    keep_sources = "--keep-sources" in sys.argv
-    main(create_zip, keep_sources)
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--skip-zip",
+        dest="skip_zip",
+        action="store_true",
+        help=(
+            "Skip zipping server package and create only"
+            " server folder structure."
+        )
+    )
+    parser.add_argument(
+        "--keep-sources",
+        dest="keep_sources",
+        action="store_true",
+        help=(
+            "Keep folder structure when server package is created."
+        )
+    )
+    parser.add_argument(
+        "-o", "--output",
+        dest="output_dir",
+        default=None,
+        help=(
+            "Directory path where package will be created"
+            " (Will be purged if already exists!)"
```

Comment:
This could be improved I think.

The output dir is not the `package` directory but the folder _in which_ the packages gets created. Right?
Also, the purging only occurs if `--clear-output-dir` is passed though. (However any packages inside the output dir to be overwritten will get purged always)